### DISCUSSION TITLE
反代始终没有成功过

### DISCUSSION THREAD

**Discussion by Doutianbao** [1Panel-dev/1Panel#5250] [Q&A]
### 1Panel 版本

v1.10.9-lts

### 问题描述

比如我docker安装这个pdf服务
![image](https://github.com/1Panel-dev/1Panel/assets/25342342/431b5f6d-5c85-4d38-9652-4be56f86a285)

### 重现步骤

反代设置为127.0.0.1:3080
pdf.xxx.com域名使用*.xxx.com申请的ssl

无法反代成功

**Comment by Doutianbao** [comment]
设置好后能正常使用1分钟作用，再刷新就报错ERR_HTTP2_PROTOCOL_ERROR

清楚代理缓存继续如上

未开启代理缓存

**Comment by MiderWong** [comment]
把WAF关了再试试

**Reply by Doutianbao** [reply]
还是不行
哭了😭

**Comment by MiderWong** [comment]
关闭WAF的情况下，我测试了`Stirling-PDF`的安装、部署、反向代理以及`Login`设置，全都没有问题，包括反向代理后的功能使用，也没有任何问题。
![image](https://github.com/1Panel-dev/1Panel/assets/7487008/003935cf-82e5-47ca-ac48-bccdd16a40e8)
![image](https://github.com/1Panel-dev/1Panel/assets/7487008/014d9126-bd51-45c3-ac97-8662c79e56df)

**Comment by Doutianbao** [comment]
谢谢您的回复，我使用的免费版本，没有开启WAF功能
Thank you for your reply, but I am not the premium version~

**Comment by MiderWong** [comment]
建议去确认一下WAF功能是否关闭，因为：
1、社区版本也提供比较完善的WAF功能
2、社区版本也会默认启动WAF功能
如果确实关闭了WAF功能，建议把 `容器内的错误日志` 和 `域名的错误日志` 贴出来共同探讨问题
